### DISCUSSION TITLE
Check List Percentages

### DISCUSSION THREAD

**Discussion by maxlingenfelter** [community/community#20813] [Codespaces]
I have been messing with GitHub project boards recently along with Trello.
One of my favorite things about Trello is its checklists it has a percentage of completion and I think this would be really nice to have on GitHub project boards.

Trello:
![image](https://user-images.githubusercontent.com/36557751/179528302-cbc1379e-6a71-4c41-ace0-6c67c891d28e.png)

Github:
![image](https://user-images.githubusercontent.com/36557751/179528425-3bdf1670-4c28-4f22-9da7-94183062bd97.png)

**Comment by thaisonzdx** [comment]
Unfortunaly, it's not available right now. But you can add multiple issues to a milestone to track percentage of your works. You can create a milestone on this link
```
https://github.com/{username}/{repository}/milestones
```
